src/frontend/src/app/project-configuration/project-configuration.component.ts

```diff
@@ -0,0 +1,74 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+import {UserDto} from '../model/userDto';
+import {ProjectConfigurationService} from './project-configuration.service';
+import {ProjectDto} from '../model/projectDto';
+import {StorageService} from '../shared/storage.service';
+import {ProjectMemberDto} from '../model/projectMemberDto';
+
+@Component({
+  selector: 'app-project-configuration',
+  templateUrl: './project-configuration.component.html',
+  styleUrls: ['./project-configuration.component.css']
+})
+
+export class ProjectConfigurationComponent implements OnInit {
+
+  users: UserDto[] = [];
+  project: ProjectDto = new ProjectDto();
+  roles: string[] = ['developer', 'tester', 'project manager'];
+  selectedRole: string;
+  error: string;
+
+  constructor(private route: ActivatedRoute,
+              private router: Router,
+              private confService: ProjectConfigurationService,
+              private storageService: StorageService) {
+  }
+
+  ngOnInit() {
+    if (!this.storageService.getScope()) {
+      this.router.navigate(['/home']);
+    }
+    this.project = this.storageService.getScope();
+    this.getAllUsers();
+    this.error = '';
+  }
+
+  getAllUsers() {
+    const members: string[] = [];
+    for (const i of this.project.members) {
+      members.push(i.username);
```

Comment:
tutaj możesz użyć map, czyli:
const members: string[] = this.project.members.map(m => m.username);